CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## [Unreleased]
 ### Changed
 - [#60] Adds layout optimizations for mobile devices.
+- [#105] Fixes an issue which prevented the address book download button to work on Firefox.
+- [#101] Fixes the reset button of the payment dialog to reset the validity and input values correctly.
```

Comment:
You should also add links in the bottom of the page so that the issues are linkable.
e.g. :`[#105]: https://github.com/raiden-network/webui/issues/105`

Reply:
Thanks!